src/Symfony/Component/Form/Extension/Core/Type/TelType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+
+class TelType extends AbstractType
```

Comment:
imo. telephones matches with tel. It's just a fully qualified reference. Imo. `TelType` is a bad class name :)